src/app/[locale]/projects/detail/[id]/components/Obligations/ObligationsView/LicenseObligation.tsx

```diff
@@ -0,0 +1,218 @@
+                            })}
+                        </ul>
+                    }
+                </div>
+            ),
+            sort: true,
+        },
+        {
+            id: 'licenseObligation.releases',
+            name: t('Releases'),
+            formatter: ({ releases, releaseIdToAcceptedCLI }: 
+                { releases: LicenseObligationRelease[],  releaseIdToAcceptedCLI: { [k: string]: string } }) =>
+            _(
+                <div className='text-center'>
+                    {Object.keys(releaseIdToAcceptedCLI).map((releaseId: string, index: number) => {
+                        const rel = releases.find((r) => r["_links"]["self"]["href"].substring(r["_links"]["self"]["href"].lastIndexOf("/") + 1) === releaseId)
+                        return (
+                            <li key={releaseId} style={{ display: 'inline' }}>
+                                <Link href={`/licenses/${releaseId}`} className='text-link'>
+                                    {`${rel.name} ${rel.version}`}
+                                </Link>
+                                {index >= Object.keys(releaseIdToAcceptedCLI).length - 1 ? '' : ', '}{' '}
+                            </li>
+                        )
+                    })}
+                </div>
+            ),
+            sort: true,
+        },
+        {
+            id: 'licenseObligation.status',
+            name: t('Status'),
+            sort: true,
+        },
+        {
+            id: 'licenseObligation.type',
+            name: t('Type'),
+            sort: true,
+        },
+        {
+            id: 'licenseObligation.id',
+            name: t('Id'),
+            sort: true,
+        },
+        {
+            id: 'licenseObligation.comment',
+            name: t('Comment'),
+            sort: true,
+        }
+    ]
+
+    useEffect(() => {
+        if (status !== 'authenticated') return
+        const controller = new AbortController()
+        const signal = controller.signal
+        
+        ;(async () => {
+            try {
+                const res = await ApiUtils.GET(
+                    `projects/${projectId}/licenseObligations`,
+                    session.user.access_token,
+                    signal
+                )
+
+                if (res.status === HttpStatus.UNAUTHORIZED) {
+                    return signOut()
+                } else if (res.status !== HttpStatus.OK) {
+                    return notFound()
+                }
+
+                const licenseObligations: ProjectObligation = await res.json()
+            
+                const releaseRequests: Promise<Response>[] = []
+                for (const obligation in licenseObligations) { 
+                    const releaseIds = Object.keys(
+                        licenseObligations[obligation].releaseIdToAcceptedCLI ?? {}
+                    )
+                    for (const releaseId of releaseIds) {
+                        releaseRequests.push(ApiUtils.GET(
```

Comment:
This may cause performance issue if we have much releases in obligation.
I think releases should be embedded in obligation response. We only need one request to GET all data